packages/front-end/pages/experiments/index.tsx

```diff
@@ -76,21 +58,16 @@ const ExperimentsPage = (): React.ReactElement => {
 
   const { templates } = useTemplates();
 
-  const [tabs, setTabs] = useLocalStorage<string[]>("experiment_tabs", []);
+  const [tab, setTab] = useState<string>("all");
```

Comment:
Do we want to do `useLocalStorage` here so it remembers the last tab the user had selected?

Reply:
I thought about that - what I'm concerned with is that you click on experiments expecting to see all experiments, but its on your last filter, which requires you to see that the old tab is selected. Open to be wrong, but I feel this is more expected behaviour.

Reply:
https://growthbookapp.slack.com/archives/C02BHUS8NBE/p1754389045626759 someone only cares about running experiments and is annoyed about having to constantly go back to that tab.   I'd have to agree that I think that is the more common use case.  People who can't find an experiment will hopefully see they are on "running experiments" or some other tab.